security/openconnect/src/opnsense/mvc/app/models/OPNsense/Openconnect/General.xml

```diff
@@ -25,7 +25,7 @@
         </password>
         <servercert type="TextField">
             <Required>N</Required>
-            <mask>/^[a-z0-9]{40,64}$/</mask>
+            <mask>/^([a-zA-Z0-9\/\+\=]){40,64}$/u</mask>
```

Comment:
in theory you don't need the round bracket, the count operator also works for the square brackets.
You can also omit the A-Z or the a-z if you add an i at the end (i means no case sensitive matching).

No need to change that - it's just a question of taste.